_data/simple-icons.json

```
@@ -10556,8 +10556,8 @@
 	},
 	{
 		"title": "MYOB",
-		"hex": "6100A5",
-		"source": "https://myob-identikit.frontify.com/d/JK2D4WFOdAwV/for-developers"
+		"hex": "7B14EF",
+		"source": "https://myob-identikit-brand.frontify.com/hub/24"
```

```suggestion
		"source": "https://www.myob.com"
```

The page you'd linked is locked behind a login, with only a PNG available on the login page. The homepage of the brand has an SVG we can use.